service/controller/app/v1/resource/chartoperator/resource.go

```diff
@@ -170,13 +170,11 @@ func (r Resource) installChartOperator(ctx context.Context, cr v1alpha1.App) err
 	var chartOperatorValue []byte
 	{
 		v := map[string]interface{}{
-			"resource": map[string]interface{}{
-				"image": map[string]string{
-					"registry": r.registryDomain,
-				},
-				"tiller": map[string]string{
-					"namespace": namespace,
-				},
+			"image": map[string]string{
+				"registry": r.registryDomain,
+			},
+			"tiller": map[string]string{
```

Comment:
After this change, tiller had been installed in `kube-system` in e2e test. Did you align it with template changes?

Reply:
https://github.com/giantswarm/chart-operator/pull/287 ?

Reply:
ah, the problem is that we are pulling older chart-operator in e2e, we need to pull the latest in this case. @rossf7 Can you update that?

Reply:
Thanks! Yes makes sense it's the tiller namespace.

But the values structure needed to change. The resource block is just for installing multiple copies of the chart for VOO.

But it's changed in master for both charts. So what chart tarball is being used?

Reply:
@tomahawk28 Damn our updates crossed! 😆 Yes sure!

Reply:
Should be good now but let's see.🤞

Reply:
I didn't change the catalog.

![picard-facepalm](https://user-images.githubusercontent.com/311527/65153810-b3b55d80-da2a-11e9-8364-7b308cc788ac.jpg)

Reply:
I think you need to change the name of here too https://github.com/giantswarm/app-operator/blob/790dcc062a9c4e79f3d900906063d6fb4a03591a/integration/test/app/kubeconfig/kubeconfig_test.go#L132

Reply:
Wow reading is hard today. Thx!